docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-cortex.md

```diff
@@ -8,11 +8,11 @@ weight: 10
 # Migrate from Cortex to Grafana Mimir
 
 As an operator, you can migrate a Helm deployment of [Cortex](https://cortexmetrics.io/) to Grafana Mimir.
-The overview includes the steps required for any environment. To migrate deployment environments with Helm, see [Migrate to Grafana Mimir using Helm]({{< relref "#migrate-to-grafana-mimir-using-helm" >}}).
+The overview includes the steps required for any environment. To migrate deployment environments with Helm, see [Migrate to Grafana Mimir using Helm](#migrate-to-grafana-mimir-using-helm).
```

Comment:
Why not a relref here?

Reply:
It does no harm to wrap it in a `relref` but it also takes extra key presses and doesn't give you any additional checks. `relref` only checks link destinations and not anchors.

Reply:
TIL :)